.cirrus.yml

```
@@ -7,12 +7,12 @@ flutter_docker_builder:
     # from https://flutter.dev/docs/development/tools/sdk/releases
     matrix:
     - DOCKER_TAG: latest
-      FLUTTER_VERSION: v1.12.13+hotfix.8
+      FLUTTER_VERSION: v1.12.13+hotfix.9
     - DOCKER_TAG: stable
-      FLUTTER_VERSION: v1.12.13+hotfix.8
+      FLUTTER_VERSION: v1.12.13+hotfix.9
     - DOCKER_TAG: beta
-      FLUTTER_VERSION: v1.15.17
+      FLUTTER_VERSION: v1.17.0-dev.3.1
```

It seems Flutter team dropped `v` from the tags from 1.17 version.

```suggestion
      FLUTTER_VERSION: 1.17.0-dev.3.1
```

---

Yep you're right, I committed a new version removing the v from beta and dev